src/shogun/mathematics/linalg/ratapprox/logdet/opfunc/LogRationalApproximationIndividual.cpp

```diff
@@ -107,7 +97,7 @@ CJobResultAggregator* CLogRationalApproximationIndividual::submit_jobs(
 	else
 	{
 		// something weird happened
-		SG_ERROR("OperatorFunction::submit_jobs(): Unknown MatrixOperator given!\n");
+		SG_ERROR("OperatorFunction::solve(): Unknown MatrixOperator given!\n");
```

Comment:
in all error messages that you touch, remove the `ClassName::method_name`, this is outdated